packages/x-components/src/views/LayoutNext.vue

```diff
@@ -0,0 +1,291 @@
+  import ChevronUp from '../components/icons/chevron-up.vue';
+  import CrossIcon from '../components/icons/cross.vue';
+  import Grid1Col from '../components/icons/grid-1-col.vue';
+  import Grid2Col from '../components/icons/grid-2-col.vue';
+  import SearchIcon from '../components/icons/search.vue';
+  import Layout from '../components/layouts/layout.vue';
+  import BaseIdModalClose from '../components/modals/base-id-modal-close.vue';
+  import BaseIdModalOpen from '../components/modals/base-id-modal-open.vue';
+  import BaseIdModal from '../components/modals/base-id-modal.vue';
+  import BaseHeaderTogglePanel from '../components/panels/base-header-toggle-panel.vue';
+  import BaseResultImage from '../components/result/base-result-image.vue';
+  import BaseScrollToTop from '../components/scroll/base-scroll-to-top.vue';
+  import SlidingPanel from '../components/sliding-panel.vue';
+  import { infiniteScroll } from '../directives/infinite-scroll/infinite-scroll';
+  import { XPlugin } from '../plugins/x-plugin';
+  import { XInstaller } from '../x-installer/x-installer';
+  import Facets from '../x-modules/facets-next/components/facets/facets.vue';
+  import SimpleFilter from '../x-modules/facets/components/filters/simple-filter.vue';
+  import MultiSelectFilters from '../x-modules/facets/components/lists/multi-select-filters.vue';
+  import HistoryQueries from '../x-modules/history-queries/components/history-queries.vue';
+  import NextQueries from '../x-modules/next-queries/components/next-queries.vue';
+  import PopularSearches from '../x-modules/popular-searches/components/popular-searches.vue';
+  import QuerySuggestions from '../x-modules/query-suggestions/components/query-suggestions.vue';
+  import Recommendations from '../x-modules/recommendations/components/recommendations.vue';
+  import RelatedTags from '../x-modules/related-tags/components/related-tags.vue';
+  import ClearSearchInput from '../x-modules/search-box/components/clear-search-input.vue';
+  import SearchButton from '../x-modules/search-box/components/search-button.vue';
+  import SearchInput from '../x-modules/search-box/components/search-input.vue';
+  import Banner from '../x-modules/search/components/banner.vue';
+  import BannersList from '../x-modules/search/components/banners-list.vue';
+  import Promoted from '../x-modules/search/components/promoted.vue';
+  import PromotedsList from '../x-modules/search/components/promoteds-list.vue';
+  import ResultsList from '../x-modules/search/components/results-list.vue';
+  import SortDropdown from '../x-modules/search/components/sort-dropdown.vue';
+  import SortList from '../x-modules/search/components/sort-list.vue';
+  import { searchXModule } from '../x-modules/search/x-module';
+  import { baseInstallXOptions, baseSnippetConfig } from './base-config';
+
+  @Component({
+    beforeRouteEnter(_to, _from, next: () => void): void {
+      XPlugin.registerXModule(searchXModule);
```

Comment:
We don't need to register the search module manually

Reply:
coming from the old version. Copy+Paste
I think there are many places were we have to remove it also